server/interceptor/interceptor.go

```diff
@@ -121,9 +126,23 @@ func getRPCInfo(req interface{}, fullMethod string) (*rpcInfo, error) {
 
 	readonly := strings.HasPrefix(methodName, "Get") || strings.HasPrefix(methodName, "List")
 
+	var kind quota.Kind
+	if readonly {
+		kind = quota.Read
+	} else {
+		kind = quota.Write
+	}
+	var specs []quota.Spec
```

Comment:
Yes, order matters (it's documented on quota.go). We don't want to take Global tokens if the User is over QPS, for example.